deploy/overlays/prod/patch_deployment.yaml

```diff
@@ -31,8 +31,13 @@ spec:
         - name: redis-config
           configMap:
             name: redis-config
+        - name: ai-config
```

Comment:
This can be removed since it's added to base

Reply:
Done.